catalog/task/get-ocp-version/0.1/get-ocp-version.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: get-ocp-version
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to collect OCP version tag from FBC fragment using `skopeo inspect`
+  params:
+    - name: snapshot
```

Comment:
@scoheb @theflockers  Can you please help me with if this `snapshot` is the right param value to collect OCP version? Or Do I need something else? Have look at the [FBC Release](https://github.com/redhat-appstudio/release-service-bundles/blob/main/catalog/pipeline/fbc-release/0.16/fbc-release.yaml) pipeline if you see a correct option for FBC fragment?

Reply:
I'll let @theflockers answer this...

Reply:
If it is the snapshot, you will want to use a workspace for it, not a string variable. The other tasks changed to this recently and can be used as an example

Reply:
That's actually correct. We need to fetch the OCP version from the snapshot built by the build pipeline. Only then we match the result of this with the `fromIndex` that comes from the releaseStrategy.